angular/projects/admin-nrpti/src/app/import/import.component.ts

```diff
@@ -155,9 +155,6 @@ export class ImportComponent implements OnInit {
   }
 
   postToApi(): Observable<any> {
-    return forkJoin(
-      this.factoryService.startTask({ dataSourceType: 'epic', recordType: 'inspection' }),
-      this.factoryService.startTask({ dataSourceType: 'epic', recordType: 'order' })
-    );
+    return this.factoryService.startTask({ dataSourceType: 'epic' });
   }
```

Comment:
I think we should still have these split up and just combine the results in the front end. I know they say they wont want just order or inspections to run by themselves now, but let's say they changed their mind on that. Does this change stop us from having that ability?

Reply:
In its current form, they would not be able to update just one record type.

But, it would be easy enough to make a small change to accept an optional parameter to specify a type.  And then it would just update that 1 type.  Then it would be able to handle both cases.